Samples/iOS-Swift/iOS-SwiftUITests/LaunchUITests.swift

```diff
@@ -87,12 +86,34 @@ class LaunchUITests: XCTestCase {
             assertApp()
         }
     }
-        
-    private func waitForExistenceOfMainScreen() {
+
+    /**
+     * We had a bug where we forgot to install the frames tracker into the profiler, so weren't sending any GPU frame information with profiles. Since it's not possible to enforce such installation via the compiler, we test for the results we expect here, by starting a transaction, triggering an ANR which will cause degraded frame rendering, stop the transaction, and inspect the profile payload.
+     */
+    func testProfilingGPUInfo() throws {
+        app.buttons["Start transaction"].tap()
+        app.buttons["ANR filling run loop"].tap()
+        app.buttons["Stop transaction"].tap()
+
+        let profileBase64DataString = try XCTUnwrap(app.textFields["io.sentry.ui-tests.profile-marshaling-text-field"].value as? NSString)
+        let profileData = try XCTUnwrap(Data(base64Encoded: profileBase64DataString as String))
+        let profileDict = try XCTUnwrap(try JSONSerialization.jsonObject(with: profileData) as? [String: Any])
+
+        let metrics = try XCTUnwrap(profileDict["measurements"] as? [String: Any])
+        let slowFrames = try XCTUnwrap(metrics["slow_frame_renders"] as? [String: Any])
+        let slowFrameValues = try XCTUnwrap(slowFrames["values"] as? [[String: Any]])
+        let frozenFrames = try XCTUnwrap(metrics["frozen_frame_renders"] as? [String: Any])
+        let frozenFrameValues = try XCTUnwrap(frozenFrames["values"] as? [[String: Any]])
+        XCTAssertFalse(slowFrameValues.isEmpty && frozenFrameValues.isEmpty)
```

Comment:
`l`: Don't we also want to assert the `screen_frame_rates` here?

Reply:
We can, yes. I suppose that would've been easy enough :)